apps/comps/components/modals/choose-agent.tsx

```diff
@@ -35,7 +36,12 @@ export const ChooseAgentModal: React.FC<ChooseAgentModalProps> = ({
   onContinue,
   competition,
 }) => {
-  const { data: compAgents } = useCompetitionAgents(competition?.id);
+  const { data: compAgents } = useQuery({
+    ...tanstackClient.competitions.getAgents.queryOptions({
```

Comment:
my AI tried to do this a few times.  I'm pretty sure its wrong, or at least not the way the rest of the code does it and seems uglier.  `enabled` should be an argument to queryOptions (at the same level as "input"), not useQuery, and we don't need the spread operator.

I'm not 100% sure thats right, but it's definitely how a lot of the other calls are set up now and we should probably figure out which is right, or if they both are still probably a good idea to standardize on one approach

Reply:
Oh weird. Yea fixing.